pkg/container/types/rowid.go

```diff
@@ -74,6 +74,15 @@ func NewRowid(blkid *Blockid, offset uint32) *Rowid {
 	return &rowid
 }
 
+func NewRowIDWithObjectIDAndBlockNum(segid *Objectid, blknum uint16, offset uint32) *Rowid {
```

Comment:
Why we cannot just return the Rowid?   There is no need for pointers.

Reply:
Should we represent all RowID, BlockID, and ObjectID with values, or just modify this function?

Reply:
I modified the Objectid parameter and return type into value.